ssha/ssm.py

```diff
@@ -6,15 +6,21 @@
 import time
 
 from botocore.exceptions import ClientError
+from cryptography.hazmat.primitives import serialization as crypto_serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.backends import default_backend as crypto_default_backend
+from os import chmod
```

Comment:
I'd prefer using `os.chmod` over a separate import, for reasons I can't really explain.